docs/book/v3/quick-start.md

```diff
@@ -108,18 +175,62 @@ $hydrators->setService('Artist', $artistHydrator);
 $delegating = new Laminas\Hydrator\DelegatingHydrator($hydrators);
 
 // Now we can use $delegating to hydrate or extract any supported object
+// Assumes that $data and Artist have already been initialised
 $array  = $delegating->extract(new Artist());
 $artist = $delegating->hydrate($data, new Artist());
 ```
 
-### Laminas\\Hydrator\\ObjectPropertyHydrator
+## Base Interfaces
+
+### ExtractionInterface
+
+```php
+<?php
+
+namespace Laminas\Hydrator;
+
+interface ExtractionInterface
+{
+    /**
+     * Extract values from an object
+     *
+     * @return mixed[]
+     */
+    public function extract(object $object) : array;
+}
+```
+
+### HydrationInterface
 
-Any data key matching a publicly accessible property will be hydrated; any public properties
-will be used for extraction.
+```php
+<?php
 
-### Laminas\\Hydrator\\ReflectionHydrator
+namespace Laminas\Hydrator;
 
-Similar to the `ObjectPropertyHydrator`, but uses [PHP's reflection API](http://php.net/manual/en/intro.reflection.php)
-to hydrate or extract properties of any visibility. Any data key matching an
-existing property will be hydrated; any existing properties will be used for
-extraction.
+interface HydrationInterface
+{
+    /**
+     * Hydrate $object with the provided $data.
+     *
+     * @param mixed[] $data
+     * @return object The implementation should return an object of any type.
+     *     By purposely omitting the return type from the signature,
+     *     implementations may choose to specify a more specific type.
+     */
+    public function hydrate(array $data, object $object);
+}
+```
+
+### HydratorInterface
+
+```php
+<?php
+
+namespace Laminas\Hydrator;
+
+interface HydratorInterface extends
+    ExtractionInterface,
+    HydrationInterface
+{
+}
+```
```

Comment:
Please remove this section because in the current version it does not help the reader.